test/library/encryption/assertion.go

```diff
@@ -10,35 +10,68 @@ import (
 
 	"github.com/coreos/etcd/clientv3"
 	"github.com/stretchr/testify/require"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes"
+
+	configv1 "github.com/openshift/api/config/v1"
 )
 
 var protoEncodingPrefix = []byte{0x6b, 0x38, 0x73, 0x00}
 
+var defaultTargetGRs = []schema.GroupResource{
+	{Group: "", Resource: "secrets"},
+	{Group: "", Resource: "configmaps"},
+}
```

Comment:
we need a constant in the operator code and reference that here.

Reply:
should e2e tests trust production code :)?

I was thinking about listing all resources (directly from etcd) and checking the prefix. We would fail if some resources have the prefix but are not on that list.

Reply:
If it doesn't trust that code, just add a test here that your list matches the prod list.

Reply:
will add the test in a separate PR.